### PR TITLE
Add directional shooting

### PR DESCRIPTION
**Pull request by vincent-truong-main**
v.0.1.0

- Added a bullet manager class to manager bullet instantiation
- modified player script to allow directional input and firing of bullet
- added bullet asset

### CODE REVIEW

File: scenes/player_2.tscn [hunk 1]
```diff
@@ -1,8 +1,68 @@
...
+			print(resources)
+			resources += 1
+	else:
+		print(body.name)
+	pass # Replace with function body.
+	
+func _on_shot_timer_timeout():
+	can_shoot = true
+"
+
```
Comment (vincent-truong-main) [inline scenes/player_2.tscn:65]: Not sure why it added this to the script. Any idesa?

Comment (stephenstrome) [reply scenes/player_2.tscn:65]: I'm not entirely sure which part you're referencing. Like you didn't add any of the lines you highlighted?

Comment (vincent-truong-main) [reply scenes/player_2.tscn:65]: Yeah I didn’t add it.

Comment (stephenstrome) [reply scenes/player_2.tscn:65]: Only part I'm familiar with is "_on_area_2d_body_entered". My guess is that the same player script is on the player_2.tscn, so therefore code added to player_1.tscn may have been replicated across. The "_on_area_2d_body_entered" part is for touching the resource scene. The rest of it looks like stuff from shooting, if it's isn't exactly like the other code you had I couldn't say what it was. I'm down to keep it for now if you are and deal with it later if need be.

Comment (vincent-truong-main) [reply scenes/player_2.tscn:65]: Yeah let’s keep it for now

**Pull request merged**